### DISCUSSION TITLE
How to update flaresolverr

### DISCUSSION THREAD

**Discussion by darlopvil** [FlareSolverr/FlareSolverr#1007] [Q&A]
Hello people,

I know this is just super stupid to ask, but after getting great help here in the very first steps to get this working, I don't want to touch anything and make the program unusable or just make it stop working...

I've read that flaresolverr doesn't have currently an auto-update function, or at least, a simple button to check if there's a new version available. This leads to manually check it here, on GitHub. My question is, what do I need to do in order to upgrade the program to a newer version. I've already downloaded the latest version 3.3.11, but what's next? May I close the program now and copy all the files from the .zip to the folder where I have all the files of my current version?

**Comment by ilike2burnthing** [comment]
Delete old version, run new version. [v3.3.12](https://github.com/FlareSolverr/FlareSolverr/releases/v3.3.12) is out now.
